doc/missions.md

```diff
@@ -0,0 +1,133 @@
+# Missions
+
+## MissionSimplePayment
+
+Basic test to complete a payment on a small network of nodes.
+
+## MissionComplexTopology
+
+Complete basic payment load generation on an asymmetric network topology.
+
+## MissionInMemoryMode
+
+Complete basic payment and Soroban load generation on a network with in-memory watcher nodes.
+
+## MissionLoadGeneration
+
+Complete extended payment load generation (sustained tx rate) on a small network of nodes.
+
+## MissionLoadGenerationWithSpikes
+
+Complete extended payment load generation (sustained tx rate with periodic spikes) on a small network of nodes.
+
+## MissionLoadGenerationWithTxSetLimit
+
+Complete extended payment load generation while triggering surge pricing on a small network of nodes.
+
+## MissionMixedImageLoadGeneration
+
+Complete extended payment load generation while triggering surge pricing with mixed version images on a small network of nodes. Tests both old image majority and new image majority.
+
+## MissionHistoryGenerateAndCatchup
+
+Test catchup to generated ledger state for each protocol version.
+
+## MissionHistoryPubnetMinimumCatchup
+
+Perform minimum catchup to Public Network. Runs core `catchup current/0` command  under the hood.
+
+## MissionHistoryPubnetRecentCatchup
+
+Perform recent catchup to Public Network. Runs core `catchup current/1001` command  under the hood.
+
+## MissionHistoryPubnetCompleteCatchup
+
+Perform full catchup to Public Network. Runs core `catchup current/max` command  under the hood.
+
+## MissionHistoryPubnetParallelCatchup
+
+Perform full catchup to Public Network, but in parallel to improve performance. Runs a mix of minimum and recent catchup commands under the hood.
```

Comment:
Parallel catchup runs in batches of 16320 ledgers at the moment. It doesn't do minimum catchup. This is from the logs - 
```
2024-02-17 19:22:41.205 +00:00 [INF] Running 3151 jobs (128-way parallel) of 250 checkpoints each, to catch up to ledger 50415103 starting from 0
2024-02-17 19:22:41.205 +00:00 [INF] Printing catchup ranges
2024-02-17 19:22:41.225 +00:00 [INF] [|"catchup"; "15103/16320"|]
2024-02-17 19:22:41.225 +00:00 [INF] [|"catchup"; "31103/16320"|]
2024-02-17 19:22:41.225 +00:00 [INF] [|"catchup"; "47103/16320"|]
2024-02-17 19:22:41.225 +00:00 [INF] [|"catchup"; "63103/16320"|]
2024-02-17 19:22:41.225 +00:00 [INF] [|"catchup"; "79103/16320"|]
2024-02-17 19:22:41.225 +00:00 [INF] [|"catchup"; "95103/16320"|]
```

Reply:
thanks, I clarified the description